README.md

```diff
@@ -1,2 +1,35 @@
 # dockerized-openQA
-Provide a docker image of package openQA
+
+Provide a docker image of package openQA and openQA-worker
+
+
+## How to for collaborators
+
+#### webui
+
+1. Build the docker image
+
+```bash
+# You may need root privileges or your user should belong to docker group.
+docker-build -t openqa-webui:4.5.1509097496.8b8e8e20 github/dockerized-openQA/webui/
```

Comment:
I think better to use tag latest , if you don't want to update this documentation every time new version of openQA is released